### ISSUE TITLE
Crashing during wars

### ISSUE THREAD

**Issue by Techiastronamo** [lucasvinbr/GTA5GangMod#16]
I'm having trouble with crashing during wars. The game works fine outside of being in combat. I've tried everything, limiting the number of gangs, limiting the number of members, I've updated every possible thing, and searched through logs and can't find what's going on. I've reinstalled the mod once, and have removed all of my other mods, but I'm still crashing when I'm in combat. 

I have the game set to high priority with automatic paging file that's around 32GB on top of ~13GB of DDR4 RAM. I've downloaded the heap adjuster to stop initial crashing with the mods I had (around 30 or so mods, all of them work together, I've spent a few hours working out incompatibilities and everything is either visual or does not interact or get in the way of any of the other mods, so this is purely a Gang and Turf issue that I can't seem to solve on my own. 

I believe there's some AI logic bugged or something, that's the only reasonable and unsolved idea from what I read in Github issues, but that was all the way in 2018 without any true solution besides reducing the # of gangs and members. I hope we can figure out a solution, because I shouldn't be crashing due to memory as I don't see any difference between 50 members and 250 members in a war, or having 12 gangs on the map.

Thanks

**Comment by lucasvinbr** [comment]
Thanks for the report hehe. I still don't know the real cause for the crashes, but here are some things I'd like to know:

Do you crash when a big fight is going on, but it's not part of a war?

Do any of the gangs involved in the crash-inducing war have Freemode peds as members?

What do you do during those wars? Do you actively shoot at the enemy or just watch the fight? Do you use the mind-control feature?

**Comment by Techiastronamo** [comment]
It's ONLY during a big fight that is part of a war. Never when it isn't.

You know what, asking me about the peds makes me think it might be an issue
with my faction. I used Michael as the member so it's a bunch of clones in
a gang, and that's when the crashing began. I did a war once with normal
peds and it didn't crash. I think this might be the issue, I'll remove
Michael's model and test it again.

During these wars, I do both, either watching and actively fighting.
Crashes on both. I haven't even had a chance to try out mind-control yet.
